doc/Design.md

```diff
@@ -289,27 +289,18 @@ that the operation is synchronous. These cases are rare, though: the operation
 has to be so fast that it's not worth allowing other code to run during the
 delay.
 
-### No External Dependencies
-
-Tock chooses to not use any external libraries for any of the crates in the
-kernel. This is done to promote safety, as auditing the Tock code only requires
-inspecting the code in the Tock repository. Tock tries to be very specific with
-its use of `unsafe`, and tries to ensure that when it is used it is clear as to
-why. With external dependencies it would be significantly more challenging to
-ensure that uses of `unsafe` are valid, particularly as external libraries
-evolve.
-
-We also realize, however, that external libraries can be very useful. Tock's
-compromise has been to pull in specific portions of libraries into the
-`libraries` folder. This puts the library's source in the same repository, while
-keeping the library as a clearly separate crate. We do try to limit how often
-this happens.
-
-In the future, we hope that `cargo` and other Rust tools make it significantly
-easier to audit and manage dependencies. For example, cargo currently has no
-mechanism to emit an error if a dependency uses `unsafe`. If new tools emerge
-that help ensure that dependent code is safe, Tock would likely be able to
-leverage external dependencies.
+### External Dependencies
+
+Generally it's best to avoid adding any external crates to the Tock project.
+
+In certain situations Tock does allow external depenencies. This is decided on
+a case by case basis. For more details on this see
+[External Dependencies](ExternalDependencies.md).
+
+The other option to add an external library is to put specific portions of
+libraries into the `libraries` folder. This puts the library's source in the
+same repository, while keeping the library as a clearly separate crate. We do
+try to limit how often this happens.
```

Comment:
Possible interop with the discussions around #3318 here? We want to make sure that licenses are compatible and it may be worth linking to that.

Reply:
I don't think we'll need to document that here. Licenses will indeed matter, but that seems like something we can handle on a per-PR basis.